scripts/copy_generated_types_from_lemmy.sh

```diff
@@ -43,12 +43,9 @@ find src/types -type f -name '*.ts' -exec sed -i 's/bigint/number/g' {} +
 # find src/types -type f -name '*.ts' -exec sed -i '' -e 's/bigint/number/g' {} \;
 
 # Parse LemmyErrorType and convert it to array.
-ALL_ERRORS=$(cat src/types/LemmyErrorType.ts | \
-perl -nle 'm/\{ \s*error:\s*"(.*?)" \}/; print $1' | \
-sed '/^[[:space:]]*$/d'| \
-sed 's/^/"/;s/$/"/' | \
-tr '\n' ',')
-echo "export let AllLemmyErrors = [$ALL_ERRORS];" > src/types/AllLemmyErrors.ts
+pnpm prettier -w src/types/LemmyErrorType.ts
+ALL_ERRORS=$(cat src/types/LemmyErrorType.ts | perl -nle 'm/\{ \s*error:\s*"(.*?)" \}/; print $1' | sed '/^[[:space:]]*$/d' | sed 's/^/"/;s/$/"/' | tr '\n' ',')
+echo "export const AllLemmyErrors = [$ALL_ERRORS];" >src/types/AllLemmyErrors.ts
```

Comment:
Should there be a space after the pipe?
```suggestion
echo "export const AllLemmyErrors = [$ALL_ERRORS];" > src/types/AllLemmyErrors.ts
```